spec/openapi.yaml

```diff
@@ -2209,11 +2213,18 @@ components:
       type: integer
       minimum: 0
       example: 853113599
+      nullable: true
     RunTrgStart:
       description: Unix timestamp when this entity was created.
       type: integer
       minimum: 0
       example: 853113599
+      nullable: true
+    RunTriggerDuration:
```

Comment:
It has to be `RunDuration` instead of `RunTriggerDuration`

Reply:
Done